src/commands/force/user/create.ts

```diff
@@ -123,9 +123,9 @@ export class UserCreateCommand extends SfdxCommand {
     }
 
     // Set the alias if specified
-    if (this.flags.setalias) {
+    if (this.flags.setalias || fields.alias) {
```

Comment:
`fields.alias` is the alias in the User metadata api, not the alias that the cli uses.  They are 2 different things.

Reply:
ahhh, I thought I missed this the first time and was adding it back 👍